src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/tests/CMakeLists.txt

```diff
@@ -1,28 +1,23 @@
-cmake_minimum_required (VERSION 3.5)
-
-project (adps2mqtt_client_module_functional_test)
+set (target_name adps2mqtt_client_module_functional_test)
```

Comment:
why target_name throughout rather than the built-in PROJECT_NAME?

Reply:
I believe we were moving everything to target_name. There was PR #514 in develop to move most everything to target_name